### PR TITLE
Schedule dependencies workflow file added

### PR DESCRIPTION
**Pull request by vtoliveira**
I did a minimal file just to install scikit-lego from time to time using schedule feature of github actions.

### CODE REVIEW

**Comment by MBrouns** [comment]
Hi there, thanks for the PR! This will be a good addition.

Some questions:
- Is it worth testing on Windows?
- Is it worth also running the test suite?

I don't necessarily want those questions to block this PR though, so if there's no definitive answer on those I'm +1 on merging this

**Comment by vtoliveira** [comment]
Actually, it is totally up to you guys. I was testing first on mac-os because of issue #376 as mentioned by @koaning. I didn't add windows because in your test-windows.yml file you tested only with python 3.6, so I don't know if there is a problem. 

But as he suggested, I made PR so you could @MBrouns make suggestions.

**Comment by koaning** [comment]
My main painpoint is that when scikit-learn releases a new version we need to hear about it from an issue. It'd be nicer if we got an email of a failed master cron job such that we can check our dependencies. I wonder about extra information that it might be able to give us ... but just the cronjob would be nice to have already.

**Comment by koaning** [comment]
Actually come to think of it, let's add windows just in case.

File: .github/workflows/schedule-dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```
Comment (koaning) [inline .github/workflows/schedule-dependencies.yml:19]: I think it is fine if we only check for 3.7 for now.

File: .github/workflows/schedule-dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        os: [macos-10.15, ubuntu-latest]
```
Comment (koaning) [inline .github/workflows/schedule-dependencies.yml:20]: Would be nice to add windows. Windows latest would be fine.

File: .github/workflows/schedule-dependencies.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install scikit-lego
+        pip freeze
```
Comment (koaning) [inline .github/workflows/schedule-dependencies.yml:34]: We should run pytest here as well.

Comment (koaning) [reply .github/workflows/schedule-dependencies.yml:34]: Dependencies might install just fine even if their APIs changed. To detect if something broke, we'd still need to run the tests.

**Comment by vtoliveira** [comment]
Sorry for the delayed answer. I've committed the changes as requested. Also, I ran locally and it went everything ok. 

@koaning @MBrouns 
https://github.com/vtoliveira/scikit-lego/actions/runs/160992788

**Comment by koaning** [comment]
looks good to me, thanks!

**Pull request merged**